### DISCUSSION TITLE
About Hosts & Visitors IP's

### DISCUSSION THREAD

**Discussion by taimadoCE** [allinurl/goaccess#2422] [Ideas]
![imagen](https://user-images.githubusercontent.com/57089805/203820891-8599d259-591a-4233-9823-5db7c4aabf38.png)

Hi, @allinurl . I think it would be more interesting to replace the horizontal axis items. In my opinion in that place should be the countries...shouldn't it? Is it difficult to implement? What dou you think about it?

**Comment by allinurl** [comment]
If I understand correctly, you're looking to correlate those visitors IPs with the corresponding countries?
